### DISCUSSION TITLE
installing more than one example same time

### DISCUSSION THREAD

**Discussion by saeedPadyab** [vercel/next.js#12185] [Help]
I need to install more than one example initially but it seems some need extra Webpack configuration that you do them internally. how to do it??
for example, I need tailwind + absolute-import. when I install with-tailwind then I have to set with-absolute-import configs manually but it doesn't work.

**Comment by leerob** [comment]
As far as I know, there isn't a way to clone multiple example repos at once. What you're looking for is essentially the plugin system, I think.

You should be able to start with one example and look at the other to add the functionality you need.

**Reply by saeedPadyab** [reply]
Thanks @leerob for your response.you are right but I started with linaria example then I wanted to add absolute-import configs manually but unfortunately, it did not work. It seems each example only works by creating its command.
